src/client/src/pages/overview/boreholeTable/bottomBarContainer.tsx

```diff
@@ -89,8 +89,9 @@ const BottomBarContainer = ({
   };
 
   const onExportMultiple = async () => {
-    const exportBoreholes = boreholes.data.filter(borehole => selectionModel.includes(borehole.id));
-    const jsonString = JSON.stringify(exportBoreholes, null, 2);
+    const selectedIds = selectionModel.map(id => Number(id));
```

Comment:
REQ getAllBoreholes returned maximal 100 Boreholes, wenn das SelectionModel aber mehr als 100 Ids enthält braucht es noch ein Feedback an den User. Ich denke aus UX Sicht wäre ein Alert mit 2 Optionen vor dem Export am besten:
 sowas in die Richtung:
 "Es können maximal 100 Bohrungen exportiert werden. Wollen Sie fortfahren?" 
 Button 1:  100 erste Bohrungen exportieren
 Button 2: Abbrechen

Reply:
Nach Absprache werde ich das im CSV Export Umsetzen